src/server/worldserver/IRCBridge/IRCBridge.cpp

```diff
@@ -256,6 +257,7 @@ void IRCBridge::Login()
 
 std::string IRCBridge::MakeStringUpper(std::string string)
 {
-    std::transform(string.begin(), string.end(), string.begin(), ::toupper);
-    return string;
+    std::string temp = string;
+    boost::to_upper(temp);
+    return temp;
```

Comment:
Why change to boost? Less dependecy without it.
Also the "string" is already a copy when passed, so no need to make temp